### PR TITLE
 DEV-3178-subject of wordfflip

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by jotapero — CHANGES_REQUESTED** [review]

**Review by jotapero — APPROVED** [review]

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/wordflip/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+## wordflip
+
+### Instructions
+
+Write a function `WordFlip()` that receives a `string` and returns it with the first and last words flipped.
+
+- Prints the Output followed by newline `\n`.
+- If the string is empty, return `Invalid Output`.
+- Ignore spaces if it's more then one space in between words and all spaces in the edge of the words or sentences.
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/wordflip/README.md:9]: It is fixed.

File: subjects/wordflip/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+## wordflip
+
+### Instructions
+
+Write a function `WordFlip()` that receives a `string` and returns it with the first and last words flipped.
```
[the remark this answers is not in the document]
Comment (tiagocollot) [reply subjects/wordflip/README.md:5]: It is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

File: subjects/wordflip/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+## wordflip
+
+### Instructions
+
+Write a function `WordFlip()` that receives a `string` and returns it in reverse order.
+
+- Prints the Output followed by newline `\n`.
```
Comment (eslopfer) [inline subjects/wordflip/README.md:7]: In the previous line you said that it returns, should it print or return?

File: subjects/wordflip/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- If the string is empty, return `Invalid Output`.
+- Ignore spaces if there are more than one space in between words and ignore all spaces at the edge of the `string`.
+
+### Expected function
+
+```go
+func WordFlip(str string) string {
```
Comment (eslopfer) [inline subjects/wordflip/README.md:14]: From this signature it appears that the function should return, but before it's asked to print.

**Review by eslopfer — CHANGES_REQUESTED** [review]

**Pull request merged**